### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DerCalli**
**Initial commit**

Package provides product properties inside category-layer of fond-of-spryker/google-tag-manager

**Plugins:**
- GoogleTagManagerCategoryProductIdAbstractPlugin
- GoogleTagManagerCategoryProductNamePlugin
- GoogleTagManagerCategoryProductNamePlugin
- GoogleTagManagerCategoryProductSkuPlugin

### CODE REVIEW

**Review by SpiderDan98 — APPROVED** [review]

File: src/FondOfSpryker/Yves/GoogleTagManagerCategoryProductConnector/Model/GoogleTagManagerCategoryProductConnectorModel.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManagerCategoryProductConnector\Model;
+
+use FondOfSpryker\Shared\GoogleTagManagerCategoryProductConnector\GoogleTagManagerCategoryProductConnectorConstants as ModuleConstants;
+use Spryker\Shared\Money\Dependency\Plugin\MoneyPluginInterface;
+
+class GoogleTagManagerCategoryProductConnectorModel implements GoogleTagManagerCategoryProductConnectorModelInterface
+{
+    protected $moneyPlugin;
```
Comment (julianhyatt) [inline src/FondOfSpryker/Yves/GoogleTagManagerCategoryProductConnector/Model/GoogleTagManagerCategoryProductConnectorModel.php:13]: check

File: src/FondOfSpryker/Yves/GoogleTagManagerCategoryProductConnector/Model/GoogleTagManagerCategoryProductConnectorModel.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    public function __construct(MoneyPluginInterface $moneyPlugin)
```
Comment (julianhyatt) [inline src/FondOfSpryker/Yves/GoogleTagManagerCategoryProductConnector/Model/GoogleTagManagerCategoryProductConnectorModel.php:18]: check

File: src/FondOfSpryker/Yves/GoogleTagManagerCategoryProductConnector/Model/GoogleTagManagerCategoryProductConnectorModel.php [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+     *
+     * @return array
+     */
+    public function getProductSku(array $params): array
+    {
+        if (!isset($params[ModuleConstants::PARAM_PRODUCT_ABSTRACT_SKU])) {
+            return [];
+        }
+
+        $sku = str_replace('ABSTRACT-', '', strtoupper($params[ModuleConstants::PARAM_PRODUCT_ABSTRACT_SKU]));
```
Comment (julianhyatt) [inline src/FondOfSpryker/Yves/GoogleTagManagerCategoryProductConnector/Model/GoogleTagManagerCategoryProductConnectorModel.php:91]: Maybe as const or config value

**Review by julianhyatt — COMMENTED** [review]
you should run phpcs

**Pull request merged**